website/views/user.py

```
@@ -41,23 +50,177 @@
+        val = obj.value
+        if val in ("True", "true", "1"):
+            return True
+        if val in ("False", "false", "0"):
+            return False
+        return val
+    except Exception:
+        logger.exception("gc_get failed for key=%s", key)
+        return default
+
+
+def gc_get(key, default=None):
```

Ruff (0.14.5)

Redefinition of unused `gc_get` from line 176

(F811)